pkg/kyma/docs/install.md

```diff
@@ -0,0 +1,108 @@
+# install
+
+## Description
+
+Installs Kyma on a running Kubernetes cluster.
+
+## Usage
+
+```
+kyma install [OPTIONS]
+```
+
+## Options
+
+| Name     | Short Name | Default value| Description|
+| ----------|---------|-----|------|
+| --release | -r ||Specifies the Kyma release or git revision to be installed. Go to the [GitHub releases page](https://github.com/kyma-project/kyma/releases) to find out more about each of the available releases, or use the revision of your choice. For example, `kyma install --release master`.|
```

Comment:
@kazydek,  the table renders somehow strangely when it comes to the first column :) Also should if some flags do not have values, we should leave them blank? Or it would be better to just write "the default value for this flag/option is..." and remove the column?

Reply:
I would have a separate column as we have in configuration docs. You can write "None" whenever is there is no default value.